From 59c627d02ca444623131c1cdeef3c2da9471fa0a Mon Sep 17 00:00:00 2001
From: Jiri Olsa <jolsa@redhat.com>
Date: Mon, 25 Aug 2014 07:50:04 -0400
Subject: [tools] perf/annotate: Fix typo

Message-id: <1408953210-26343-117-git-send-email-jolsa@redhat.com>
Patchwork-id: 88820
O-Subject: [PATCH RHEL7.1 BZ1133083 116/322] perf annotate: Fix typo
Bugzilla: 1133083
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

From: Jiri Olsa <jolsa@kernel.org>

Bugzilla: 1133083
https://bugzilla.redhat.com/show_bug.cgi?id=1133083

upstream
========
commit 6f1d0c866261ec1e42850b2bf760484c100b6dd6
Author: Dongsheng Yang <yangds.fnst@cn.fujitsu.com>
Date: Mon Dec 9 17:47:48 2013 -0500

description
===========
A typo in comment of builtin-annotate.c about 'that'.
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/tools/perf/builtin-annotate.c b/tools/perf/builtin-annotate.c
index 4087ab1..6fd52c8 100644
--- a/tools/perf/builtin-annotate.c
+++ b/tools/perf/builtin-annotate.c
@@ -373,7 +373,7 @@ int cmd_annotate(int argc, const char **argv, const char *prefix __maybe_unused)
 
  if (argc) {
   /*
-   * Special case: if there's an argument left then assume tha
+   * Special case: if there's an argument left then assume that
    * it's a symbol filter:
    */
   if (argc > 1)
-- 
1.7.1